Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -262,6 +262,31 @@ private extension WorldScaleSceneView {
     }
 }
 
+public extension WorldScaleSceneView {
+    /// Determines the scene point for the given screen point.
+    /// - Parameter screenPoint: The point in screen's coordinate space.
+    /// - Returns: The scene point corresponding to screen point.
+    private func arScreenToLocation(screenPoint: CGPoint) -> Point? {
```

Comment:
According to the doc, this method is only used for the identifying workflow: https://developers.arcgis.com/ios/scenes-3d/display-scenes-in-augmented-reality/#identify-real-world-and-in-scene-objects

So I didn't expose it to the user, but rather provided a custom tap modifier.